lib/tasks/fix_fdw_credentials.rake

```diff
@@ -0,0 +1,35 @@
+desc 'Change FDW server and user credentails accroding to dot env file'
+task fix_fdw_credentials: :environment do
+  fix_primary_server_fdw_credentials
+  fix_remote_server_fdw_credentials
+end
+
+def fix_primary_server_fdw_credentials
+  config = ActiveRecord::Base.configurations['secondbase'][Rails.env]
+  server = 'fis'
+  active_record_execute(alter_fdw_server_query(server, config))
+  active_record_execute(alter_fdw_user_query(server, config))
+end
+
+def fix_remote_server_fdw_credentials
+  config = ActiveRecord::Base.configurations[Rails.env]
+  server = 'openhub'
+  second_base_execute(alter_fdw_server_query(server, config))
+  second_base_execute(alter_fdw_user_query(server, config))
+end
+
+def alter_fdw_server_query(server_name, config)
+  "ALTER SERVER #{server_name} OPTIONS(set host '#{config['host']}', set dbname '#{config['database']}', set port '#{config['port']}')"
+end
+
+def alter_fdw_user_query(server_name, config)
+  "ALTER USER MAPPING FOR #{config['username']} SERVER #{server_name} OPTIONS(SET password '#{config['password']}')"
```

Comment:
No @PDegenPortnoy, we don't have a local development database. We are using stagingDB for development. 

In the above steps, i mentioned to create development database in local server because when running rake task for test environment, it fails due to non existence of development database. Therefore, i mentioned to create development database first, followed by test database. By this approach we can run rake task without any issues. Later, we can change the **.env** file to point to stagingDB.

When writing this, one basic thing strike me. Why rake task process the factory files?. Now, i  got the root cause of this problem. 

**test/factories/jobs.rb**
```ruby
FactoryGirl.define do
  2   factory :job do
  3     slave Slave.where(hostname: Socket.gethostname).first_or_create!
  4     association :code_location
  5     status Job::STATUS_SCHEDULED
  6   end
  7  end
```
In the above code at line number 3. We are creating *slave* object on the fly instead of using association. Therefore, while loading this factory file, it throws an error. 

I fixed this problem. Now, we no need to create development database in localhost for running test case:)

### Steps to run test case
```ruby
RAILS_ENV=test rake db:create
RAILS_ENV=test rake db:structure:load
RAILS_ENV=test rake db:migrate
RAILS_ENV=test rake fix_fdw_credentials
```